test/test_oxdna/test_nucleotide.py

```diff
@@ -1,15 +1,26 @@
 from origamiUROP.oxdna import Nucleotide
 import numpy as np
 
+
 def test_Nucleotide():
     nucleotide = Nucleotide(
-        'A', 
-        np.array([1., 0., 0.]),
-        np.array([1., 0., 0.]),
-        np.array([0., 0., 1.])
+        "A",
+        np.array([1.0, 0.0, 0.0]),
```

Comment:
`1.` is the same as `1.0` - I prefer not to use `1.0` because it looks messy - not a big issue

Reply:
I'm pretty sure VSCode automatically changed this, I'll look into how to stop that happening. Wasn't intentional!

Reply:
@debeshmandal is there a particular formatting provider you use, e.g. black, autopep8, yapf, etc.